src/services/qrcode/__tests__/index.test.unit.ts

```diff
@@ -1,17 +1,11 @@
-import { createHmac } from 'crypto';
 import * as QRCode from '..';
-import { getConfig } from '../../../../config';
 
 
 describe('QRCode Service', () => {
-  const config = getConfig(process.env.NODE_ENV);
-
   describe('create', () => {
     test('Creates a signed HMAC and corresponding QR code data URL', async () => {
       const raw = 'foo';
-      const { payload, dataUrl } = await QRCode.create(raw);
-
-      expect(payload).toBe(createHmac('sha256', config.qrcode.secret).update(raw).digest('hex'));
+      const dataUrl = await QRCode.create(raw);
```

Comment:
What was the hmac doing previously? Is this test still correctly named?

Reply:
No, the test name should be changed, thanks.